pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -962,3 +951,21 @@ func trySendResult(ctx context.Context, user foundUser, foundUsersCh chan<- foun
 		return
 	}
 }
+
+func tupleConditionMet(ctx context.Context, reqCtx *structpb.Struct, typesys *typesystem.TypeSystem, t *openfgav1.TupleKey) (bool, error) {
```

Comment:
Exactly the same as https://github.com/openfga/openfga/blob/0a610a9607562a3fab5ffe02f0a297a5b8041c98/internal/graph/check.go#L843C1-L862C2.

We're definitely missing some abstraction here. ListObjects has similar code, too. https://github.com/openfga/openfga/blob/e20b7872fd20d7163ba9665b5896a17ee57dc904/pkg/server/commands/reverseexpand/reverse_expand.go#L529-L546